internal/pkg/logger/logger.go

```diff
@@ -0,0 +1,29 @@
+// package logger provides a better defaults, configuration wrappers and utility functions for slog package.
+package logger
+
+import (
+	"log/slog"
+	"os"
+)
+
+type Config struct {
+	Level        slog.Level
+	Service      string
+	VCSRevision  string
+	VCSTag       string
+	VCSBuildTime string
```

Comment:
why we can't use here VCSInfo?

Reply:
I've extracted this configuration to separate struct.